ci-operator/step-registry/osd-ccs/fleet-manager/osd-ccs-fleet-manager-workflow.yaml

```diff
@@ -0,0 +1,9 @@
+workflow:
+  as: osd-ccs-fleet-manager
+  steps:
+    pre:
+    - ref: osd-ccs-fleet-manager-sc-provision
+    post:
+    - ref: osd-ccs-fleet-manager-sc-deprovision
+  documentation: |-
+    osd ccs fleet manager
```

Comment:
The description should be `The workflow installs a service cluster and a management cluster with the specific configuration. After finishing testing, both of the two clusters will be deprovsioned.`